src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/parameters/Base64EncodedStringParameterValue/Base64EncodedStringParameterValue.jelly

```diff
@@ -27,6 +27,9 @@ THE SOFTWARE.
         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
         xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
         <f:entry title="${it.name}" description="${it.description}">
+            <div name="parameter" description="${it.description}">
```

Comment:
@conan-z Thanks for example. I confirmed both of them have either `value.jelly` or `value.groovy` in the resource directory for `ParameterValue` class.

* In [matrix-combinations-plugin](https://github.com/jenkinsci/matrix-combinations-plugin/blob/master/src/main/resources/hudson/plugins/matrix_configuration_parameter/MatrixCombinationsParameterValue/value.groovy)
* In [promoted-builds-plugin](https://github.com/jenkinsci/promoted-builds-plugin/blob/master/src/main/resources/hudson/plugins/promoted_builds/parameters/PromotedBuildParameterValue/value.jelly)

So I think you should add dedicated page resource file instead renaming.

> A parameter provider class is needed in rebuild too in order to support Gerrit Trigger plugin.

If you are correct, you must show the reason why Rebuild Plugin can deal with [NodeParameterValue](https://github.com/jenkinsci/rebuild-plugin/blob/master/src/main/resources/com/sonyericsson/rebuild/RebuildAction/NodeParameterValue.jelly) correctly though [NodeLabel Parameter Plugin](https://github.com/jenkinsci/promoted-builds-plugin) which provides [NodeParameterValue](https://github.com/jenkinsci/nodelabelparameter-plugin/blob/master/src/main/java/org/jvnet/jenkins/plugins/nodelabelparameter/NodeParameterValue.java) does not have any [RebuildParameterProvider](https://github.com/jenkinsci/rebuild-plugin/blob/master/src/main/java/com/sonyericsson/rebuild/RebuildParameterProvider.java) extended class.

Certainly, Rebuild Plugin's implementation for `NodeParameterValue` is not enough because type information is lost. But I don't agree with you as of now because I think the reason you mentioned is not enough as well.